src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/ModelStateDictionary.cs

```diff
@@ -180,7 +180,7 @@ public bool IsValid
         /// </summary>
         /// <remarks>
         /// This method allows adding the <paramref name="exception"/> to the current <see cref="ModelStateDictionary"/>
-        /// when <see cref="ModelMetadata"/> is not available or the exact <paramref name="exception"/> 
+        /// when <see cref="ModelMetadata"/> is not available or the exact <paramref name="exception"/>
```

Comment:
Line no. 292 in this file is referencing PropertyName, I suppose it needs to be changed?